app/bitbox02-api.d.ts

```diff
@@ -94,7 +99,7 @@ declare module 'bitbox02-api' {
       inputs: CardanoInput[]
       outputs: CardanoOutput[]
       fee: string
-      ttl: string
+      ttl?: string
```

Comment:
```suggestion
      ttl: string | null
```
this would be consistent with `validityIntervalStart` field type, or is it not possible for some reason?

Reply:
good catch. it is possible - the upstream package is still not typed and both null and undefined is accepted. 

maybe changing both to undefined is better (for both), as then the fields can be missing from the object, letting the library user specify only the fields they need.